### DISCUSSION TITLE
An error occurred when running webxr, please help

### DISCUSSION THREAD

**Discussion by suzhouhu** [cefsharp/CefSharp#3330] [General]
I used cefsharp86.0.241.0 to make a browser of my own, but I can’t use webxr functions, such as opening the following webpage: https://immersive-web.github.io/webxr-samples/immersive-vr-session.html when I click the Enter XR button Does the prompt "XRSession Creation failed" require additional settings? Please give a reminder from friends who have used it, thank you very much! ! (I am using a vive helmet-mounted VR device, which is normally used under Google Chrome!)

**Comment by amaitland** [comment]
Only suggestion i have is make sure you have enabled WebRTC. 

You can try asking on https://magpcss.org/ceforum/index.php to see if anyone else has tried. It's an experimental API and may require CEF to add support.

**Comment by suzhouhu** [comment]
Thank you very much for your answer. Since I have just contacted programming, I have to ask you, how can I enable WebRTC






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "cefsharp/CefSharp"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年12月20日(星期天) 凌晨3:38
收件人:&nbsp;"cefsharp/CefSharp"<CefSharp@noreply.github.com&gt;;
抄送:&nbsp;"刘鹄中"<767542048@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [cefsharp/CefSharp] cefsharp86 An error occurred when running webxr, please help (#3330)





 
Only suggestion i have is make sure you have enabled WebRTC.
 
You can try asking on https://magpcss.org/ceforum/index.php to see if anyone else has tried. It's an experimental API and may require CEF to add support.

**Reply by amaitland** [reply]
Please stop creating new issues, your only creating unnessicary work.

You can test with https://github.com/cefsharp/CefSharp.MinimalExample
WebRTC is enabled in the examples

**Reply by amaitland** [reply]
Asked at https://magpcss.org/ceforum/viewtopic.php?f=18&t=18080
